submissions/QuickPerformance/QuickPerformance.md

```diff
@@ -0,0 +1,84 @@
+## Quick Performance Check <author>oof2win2</author>
+
+If you don't have a state-of-the-art computer, you probably have seen a message saying that you were dropped from the game because your network or computer was not fast enough. Well, what does that actually mean? What can you do about it in the short and long term?
+
+### Network Speed Issues
+
+Sometimes, your network may not be the best. That can be due to many things itself, such as maintenance from your ISP or a power outage of your DNS's datacenters. There are a few utilities that can however help mitigate these sorts of issues.
+
+To explain what Factorio requires, that is a long*er* process. However, *cube* has explained it largely in [FFF#136](https://factorio.com/blog/post/fff-136). Wube have done non-standard things to let the game work, such as re-implementing large parts of TCP in UDP (TCP and UDP are types of communication of packets. TCP makes sure that everything is confirmed and arrives properly, UDP basically "doesn't care". Common UDP uses are in torrenting, as you can get other parts of files later on. TCP is used for many day-to-day things, such as web browsing) so the player wouldn't need to have two ports opened and NAT punching would work (NAT punching is a thing that is used for your router/modem to send Factorio traffic properly, explained well in [FFF#143](https://factorio.com/blog/post/fff-143))
```

Comment:
Could you split this paragraph into two and extend it a bit? The FFF #136 stuff is really interesting and though it's a lil nerdy I think some people would like to learn a bit more about it. Maybe split it into the first paragraph about TCP vs UDP as an educational section then a second paragraph more specific to Factorio and its setup?

Reply:
Couldn't fit stuff into two, so I put in three.